public/app/types/serviceaccount.ts

```diff
@@ -0,0 +1,96 @@
+import { OrgRole, Unit } from '.';
+import { SelectableValue } from '@grafana/data';
+
+export interface OrgServiceaccount {
```

Comment:
```suggestion
export interface OrgServiceAccount {
```

and please apply this change throughout the code you added.

Reply:
Will make these changes in a commit for all of the name changes